python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -533,15 +666,21 @@ async def do_enableCorrection(self, id_data):
                         self.log.exception(e)
                     else:
                         raise e
-                # Set pressure to zero.
 
+                # Set pressure to zero as this is not currently functional due to hardware
```

Comment:
Actually this is not what is happening here. I set the pressure to zero before opening the valve just so it won't create a spike in pressure in case it has a large set value.

Reply:
I don't understand.. the m2 valve is being opened before this pressure is being set... is this a bug then?

Reply:
If you send the command to set pressure with the valve closed it rejects it. So, I have to open the valve and then set the pressure to zeros. Same for m1.